director/director.go

```diff
@@ -148,11 +159,61 @@ func queryOrigins(ctx *gin.Context) {
 	ctx.JSON(http.StatusOK, res)
 }
 
+func handleFilterServer(ctx *gin.Context) {
+	sn := strings.TrimPrefix(ctx.Param("name"), "/")
+	if sn == "" {
+		ctx.JSON(http.StatusBadRequest, gin.H{"error": "name is a required path parameter"})
+		return
+	}
+	filtered, ft := checkFilter(sn)
```

Comment:
Personal preference, so feel free not to implement, but renaming `ft` to `filterType` makes it so I don't have to hunt for what this is supposed to be the first time I see it.